EKF/control.cpp

```diff
@@ -730,9 +730,11 @@ void Ekf::controlHeightFusion()
 				}
 			}
 
-		} else if (_baro_data_ready && !_baro_hgt_faulty && !_in_range_aid_mode) {
+		} else if (_baro_data_ready && !_baro_hgt_faulty &&
```

Comment:
I still don't understand why any check regarding the range sensor is needed in this if statement.
Whether the ranger can be used is checked in the if statement above. Now we're just trying to figure out if baro can be used. This has nothing to do with the range sensor or the range aid feature.
What situation are you taking care of with this?